style.css

```diff
@@ -0,0 +1,1014 @@
+  width: 100%;
+}
+
+nav {
+  width: 100%;
+  padding: 17.5px 5%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: transparent;
+}
+
+nav img,
+.menu-items {
+  display: none;
+}
+
+nav .material-symbols-outlined {
+  font-size: 40px;
+  color: var(--primary);
+}
+
+/* Popup-nav */
+
+.popup-nav {
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(62, 53, 53, 0.861);
+  backdrop-filter: blur(8px);
+}
+
+.popup-nav-visible {
+  display: flex;
+}
+
+.popup-header {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.popup-menu {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5%;
+  width: 100%;
+}
+
+.popup-menu li {
+  width: 100%;
+  padding: 10% 28px;
+  list-style: none;
+}
+
+.popup-menu li a {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  color: var(--light);
+  text-decoration: none;
+}
+
+.popup-menu li a:focus {
+  color: var(--secondary);
+}
+
+.cancel-button {
+  margin: 16px 38px;
+  background-color: transparent;
+  border: none;
+}
+
+/* headings container */
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url("./images/main-background.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  padding: 4% 6%;
+}
+
+.starter-text {
+  font-weight: normal;
+  font-size: 25px;
+  color: var(--secondary);
+}
+
+.title {
+  margin-top: 1%;
+  font-weight: 900;
+  font-size: 32px;
+  background-image: url("images/1494.jpg");
+  color: transparent;
+  background-size: cover;
+  -webkit-background-clip: text;
+}
+
+.disc-container {
+  max-width: 330px;
+  background-color: #f7f7f7;
+  border: 4px solid #fff;
+  padding: 4%;
+  margin-top: 4%;
+}
+
+.disc-container p {
+  font-weight: 400;
+  font-size: 12px;
+  color: #515151;
+}
+
+.date-text {
+  font-size: 20px;
+  font-weight: 900;
+  color: #595959;
+  margin-top: 6%;
+}
+
+.company {
+  font-size: 13px;
+  font-weight: 700;
+  margin-top: 2%;
+  color: #868686;
+  margin-bottom: 2%;
+}
+
+/* program-list */
+
+.program-list {
+  background-color: var(--primary);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-inline: 2%;
+  width: 100%;
+}
+
+.program-list h2 {
+  color: #fff;
+  margin-top: 7%;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.program-list hr {
+  width: 10%;
+  height: 2px;
+  background-color: #f8aca2;
+  margin-top: 3%;
+  border: none;
+}
+
+.event-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  margin-top: 9%;
+}
+
+.event-card {
+  display: grid;
+  justify-content: space-between;
+  align-items: center;
+  grid-template-columns: 18% 18% 50%;
+  gap: 10px;
+  padding: 2%;
+  background-color: #3b3e45;
+}
```

Comment:
- [ ] Please increase the column size of the `h3` element of `.event-card` as some of the headings are overflowing to the `p` section

<img width="368" alt="Screenshot 2023-03-18 at 12 45 02 PM" src="https://user-images.githubusercontent.com/44798044/226094381-b59beb77-0c82-4cd1-b9c5-446fd61affa4.png">